src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/pin-number/page.tsx

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { HighRiskBreachLayout } from "../HighRiskBreachLayout";
+
+export default function PinNumberDataBreach() {
+  return (
+    <div>
+      <HighRiskBreachLayout dataBreach="PIN" />
```

Comment:
Q: SHould this be "PIN" or "pin"?
I know it's an acronym, but maybe feels inconsistent w/ `dataBreach="creditCard"`